### PR TITLE
[SPIDER] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

**Comment by AndreiMiloiu** [comment]
Any news?

File: base/applications/games/spider/lang/ro-RO.rc [hunk 1]
```diff
@@ -27,20 +27,20 @@ BEGIN
...
-    DEFPUSHBUTTON "Con&firmă", IDOK, 114, 165, 54, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 176, 165, 54, 14
+    DEFPUSHBUTTON "OK", IDOK, 114, 165, 54, 14
+    PUSHBUTTON "Revocare", IDCANCEL, 176, 165, 54, 14
 END
 
 IDD_DIFFICULTY DIALOGEX 100, 100, 106, 80
 STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_SHELLFONT
-CAPTION "Joc la categoria"
+CAPTION "Dificil"
```
Comment (GeoB99) [inline base/applications/games/spider/lang/ro-RO.rc:36]: Dificil? Perhaps using "selectare dificultate" or something similar fits better.

Comment (AndreiMiloiu) [reply base/applications/games/spider/lang/ro-RO.rc]: It's fixed now. Now there is Selectare nivel de dificultate.

**Comment by AndreiMiloiu** [comment]
Any news?

**Pull request merged**